doc/charm++/manual.rst

```diff
@@ -9792,6 +9942,47 @@ to the Charm++ build system:
    run ``make charm++`` to compile Charm++ which includes the new load
    balancing strategy files.
 
+Writing a new load balancing strategy with TreeLB
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Writing a load balancing strategy with TreeLB is very simple. It involves
+implementing a class inheriting from the abstract `Strategy` class
+defined in the `TreeStrategyBase.h` header file. Specifically, the
+load balancing strategy needs to be implemented in the `solve` method,
+which receives objects and processors as vectors. The solution
+needs to be stored in the `solution` object and is used by the load
+balancing framework to perform object migrations accordingly. The
+`bool objSorted` argument specifes whether the incoming vector of
+objects is sorted by load values or not. This new strategy can written
+in a header file and included in the `TreeStrategyFactory.h` header
+file along with the mapping of the strategy name and class to be
+invoked for the newly implemented strategy.
+
+.. code-block:: c++
+
+  namespace TreeStrategy
+  {
+  template <typename O, typename P, typename S>
+  class FooLB : public Strategy<O, P, S>
+  {
+  public:
+    void solve(std::vector<O>& objs, std::vector<P>& procs, S& solution, bool objsSorted)
+    {
+      int index = 0;
+      for (const auto& o : objs)
+      {
+        P p = procs[index];
+        /// The strategy goes here
+        /// The strategy goes here
+        /// The strategy goes here
```

Comment:
Is there a way you could describe a simple strategy here (e.g. something like a RandomLB)? It's not clear which of the variables need to be modified here.